HW_1/Task2.py

```diff
@@ -0,0 +1,12 @@
+# Вместе они сделали S журавликов. Сколько журавликов сделал каждый ребенок,
+# если известно, что Петя и Сережа сделали одинаковое количество журавликов,
+# а Катя сделала в два раза больше журавликов, чем Петя и Сережа вместе?
+
+# *Пример:*
+# 6 -> 1  4  1
+# 24 -> 4  16  4
+# 60 -> 10  40  10
+
+s = int(input('Введите число сделанных журавликов: '))
+
+print((s//6), ((s//6)*4), (s//6))
```

Comment:
У вас трижды повторяется одно и то же действие, чтобы такого не происходило используют переменные.
```suggestion
var = s // 6
print(var , var  * 4, var)
```